var/spack/repos/builtin/packages/mkl/package.py

```diff
@@ -0,0 +1,66 @@
+from spack import *
+import os, re
+
+class Mkl(Package):
+    """Intel Math Kernel Library.
+
+    Note: You will have to add the download file to a
+    mirror so that Spack can find it. For instructions on how to set up a
+    mirror, see http://software.llnl.gov/spack/mirrors.html"""
+
+    homepage = "https://software.intel.com/en-us/intel-mkl"
+
+    # TODO: can also try the online installer (will download files on demand)
+    version('11.3.2.181', '536dbd82896d6facc16de8f961d17d65',
+            url="file://%s/l_mkl_11.3.2.181.tgz" % os.getcwd())
+
+    #provides('mkl')
```

Comment:
shouldn't this be `provides('blas') + provides('lapack')` ? together with something like
```
def setup_dependent_package(self, module, dspec):
       libdir = find_library_path('libmkl_rt.%s' % dso_suffix, self.prefix.lib64, self.prefix.lib)
       # TODO: what to do with static libs?
       # self.spec.blas_static_lib   = join_path(libdir, 'libopenblas.a')
       # self.spec.lapack_static_lib = self.spec.blas_static_lib

       # For now to make life easier stick with run-time MKL library. 
       # Otherwise we would need the whole list of libraries... :-(
        if '+shared' in self.spec:
            self.spec.blas_shared_lib   = join_path(libdir, 'libmkl_rt.%s' % dso_suffix)
            self.spec.lapack_shared_lib = self.spec.blas_shared_lib
```